charts/tensorleap/charts/node-server/templates/deployment.yaml

```diff
@@ -28,7 +28,7 @@ spec:
         - name: db-migration
           securityContext:
             runAsUser: 0
-          image: docker.io/bitnami/mongodb:5.0.9-debian-11-r3
+          image: docker.io/bitnami/mongodb:6.0.0-debian-11-r0
```

Comment:
the image could be declared in helm values so future changes will be set in one place, this will reduce the chances of differents mongo images